alot/walker.py

```diff
@@ -6,6 +6,11 @@
 import urwid
 
 
+def chain(*iterables):
```

Comment:
Is there a reason we can not use `itertools.chain` for this?

Reply:
Good point. I just didn't know it existed. I'm using `itertools.chain` now.